CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New Features
 
+  * [#782](hhtps://github.com/toptal/chewy/pull/782): Wrapped `Elasticsearch::API::Indices::Actions#clear_cache` with `.clear_cache` in Index class ([@Vitalina-Vakulchyk][])
```

Comment:
Sounds like a change to me, not a new feature.

Reply:
@bbatsov Thank you! I added this change.